helm/adaptdl-sched/templates/config.yaml

```diff
@@ -10,3 +10,6 @@ data:
   {{- if .Values.job.defaultResources }}
   ADAPTDL_JOB_DEFAULT_RESOURCES: {{ .Values.job.defaultResources | toJson | quote }}
   {{- end }}
+  {{- if .Values.job.jobPatch }}
+  ADAPTDL_JOB_POD_PATCH : {{ .Values.job.jobPatch | toJson | quote }}
+  {{- end }}
```

Comment:
Split into two env vars: `ADAPTDL_JOB_PATCH_PODS` and `ADAPTDL_JOB_PATCH_CONTAINERS` referring to `.Values.job.patch.pods` and `.Values.job.patch.containers`, respectively.